common/web/lm-worker/src/test/test-runner/web-test-runner.config.mjs

```diff
@@ -0,0 +1,57 @@
+// @ts-check
+import { devices, playwrightLauncher } from '@web/test-runner-playwright';
+import { defaultReporter, summaryReporter } from '@web/test-runner';
+import { esbuildPlugin } from '@web/dev-server-esbuild';
+import named from '@keymanapp/common-test-resources/test-runner-rename-browser.mjs'
+import { fileURLToPath } from 'url';
+import { dirname, resolve } from 'path';
+import { LauncherWrapper, sessionStabilityReporter } from '@keymanapp/common-test-resources/test-runner-stability-reporter.mjs';
+
+const dir = dirname(fileURLToPath(import.meta.url));
+const KEYMAN_ROOT = resolve(dir, '../../../../../../');
+
+/** @type {import('@web/test-runner').TestRunnerConfig} */
+export default {
+  // debug: true,
+  browsers: [
+    new LauncherWrapper(playwrightLauncher({ product: 'chromium' })),
+    new LauncherWrapper(playwrightLauncher({ product: 'firefox' })),
+    new LauncherWrapper(playwrightLauncher({ product: 'webkit', concurrency: 1 })),
+    named(new LauncherWrapper(playwrightLauncher({ product: 'webkit', concurrency: 1, createBrowserContext({browser}) {
```

Comment:
I've been running into issues with testing against the WebKit engine when concurrency is set greater than 1.  I don't think I actually had that problem for _this_ test set, in particular, but I set the workaround in place just the same.